### DISCUSSION TITLE
'npm run deploy' runs forever

### DISCUSSION THREAD

**Discussion by generic-matrix** [mozilla/hubs#3827] [General Questions & Discussion]
**Description**
When we run 'npm run deploy' it says deploying but it takes forever

**To Reproduce**
1) Followed the docs : https://hubs.mozilla.com/docs/hubs-cloud-custom-clients.html

**Expected behavior**
To deploy 

**Screenshots**
![FL-25-06-2020-09-44-48](https://user-images.githubusercontent.com/15347450/85759818-e2d1cb80-b72e-11ea-9e1e-442bff28b434.jpeg)

![FL-25-06-2020-09-45-56](https://user-images.githubusercontent.com/15347450/85759788-dcdbea80-b72e-11ea-8a67-31c10536e281.jpeg)

**Hardware** 
-Ubuntu 16.0 LTS AWS ec2

**Comment by InfiniteLee** [comment]
@generic-matrix It looks like you're getting an error in your second screenshot because you don't have `make` installed. Try installing `build-essential` via `apt` and try again.

**Comment by zhaozj89** [comment]
What is the normal time after running `npm run deploy`? I have waited for nearly one day, but it still says building client.

<img width="720" alt="Screenshot 2020-08-12 at 4 28 07 PM" src="https://user-images.githubusercontent.com/34164876/89993392-3a190480-dcb9-11ea-8d19-ba339c252a9f.png">

**Reply by pattersonbl2** [reply]
Following back on this questions. My own deployment took just a few mins.  Are you still have a issues with this ?
